src/configuration/ConfigEditor.tsx

```diff
@@ -24,6 +25,36 @@ export const ConfigEditor = (props: Props) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const [saved, setSaved] = useState(!!options.version && options.version > 1);
+  const datasource = useDatasource(props);
+
+  useEffect(() => {
+    setSaved(false);
+  }, [
+    options.url,
```

Comment:
This is probably excessive, but since the user has to click the button to start saving it, I figured it was better to err on the side of re-saving too often.